### DISCUSSION TITLE
Osmosis：Personal boundaries have the properties of a semi-permeable membrane.[Hong Yu]

### DISCUSSION THREAD

**Discussion by Axiuuuuu** [realitydeslab/playshop-summer-2024#19] [Proposal]
### **Chinese Title:**
透溶界——具有半透膜性质的个人边界泡泡

### **Your Team:**
Hong Yu

### **Research Questions（研究问题）**：
在虚拟空间（元宇宙）中，人与人交往的边界如何界定？我们应该通过何种形式进行规范与许可？
In the virtual space , how to define the boundaries of human interaction? In what form should we regulate and license?

### **Keywords（关键字）**：
去中心化，巴赫金人学思想，个人边界，零知识证明，HRV，情绪检测
Decentralization, Bakhtin’s humanistic thought, personal boundaries, zero-knowledge proof，HRV，Emotion detection;

### **Artefact / Media（工艺品/媒体）**：
问卷，视频，模型
Questionnaire,Video,Model

### **Abstract（概念）**：

After the popularization of virtual reality technology in the future, not only the visual senses, but also all other senses will become freely definable with the development of technology. Unlike the real world we are exposed to now, the future world is a decentralized world where ownership is distributed between its builders and users. Everyone has equal access rights to apply for access to other people's spaces, and the permission rights are fully controlled by the space owner.
在未来的虚拟现实技术普及后，不仅视觉感官，其他所有感官也将随着技术的发展而变得可以自由定义。与我们现在所接触的现实世界不同，未来世界是一个去中心化的世界，所有权在其构建者和用户之间分配。每个人都有平等的访问权限去申请访问别人的空间，允许权由空间主人全权掌握。

“透溶界”泡泡**是一个未来的概念，描绘了在虚拟现实和去中心化技术普及后，个人边界如何变得可变和自适应。在这个去中心化的世界里，“透溶界”泡泡不仅是个人隐私和安全的保护机制，更是个人表达和社交互动的工具。

**Permeable Boundary Bubble** is a futuristic concept depicting how personal boundaries become adaptable and flexible with the proliferation of virtual reality and decentralized technologies. In this decentralized world, the "Permeable Boundary Bubble" is not only a mechanism for protecting personal privacy and security but also a tool for personal expression and social interaction.

个人边界泡泡在这样的系统中发挥了重要作用：
The personal boundary bubble plays an important role in such a system:

1. **隐私保护**：泡泡的溶透性和大小可以随主人的意愿调整，保护个人隐私。
   **Privacy Protection**: The permeability and size of the bubble can be adjusted according to the owner's will, protecting personal privacy.

2. **个性表达**：泡泡的外观和材质变化成为个人风格的一部分，表达主人的心情和状态。
   **Personal Expression**: The appearance and material of the bubble can change, becoming part of the individual's style and expressing the owner's mood and state.

3. **社交互动**：用户之间的泡泡可以相融或弹开，以表达接纳或拒绝。
   **Social Interaction**: The bubbles between users can merge or bounce apart, expressing acceptance or rejection.

4. **访问控制**：用户通过设置准入条件管理谁可以进入他们的空间，确保访问者符合期望。
   **Access Control**: Users can manage who can enter their space by setting access conditions, ensuring that visitors meet expectations.

5. **情感和状态反馈**：泡泡根据主人的情感和状态自动调整，作为非语言的交流方式。
   **Emotional and State Feedback**: The bubble automatically adjusts based on the owner's emotions and state, serving as a non-verbal communication method.

6. **安全和防护**：保护主人免受潜在威胁，检测到威胁时自动加强防护。
   **Safety and Protection**: Protects the owner from potential threats, automatically strengthening protection when a threat is detected.

7. **自我调节**：连接主人的生理和心理状态，自我调节以提供最佳的舒适度和安全感。
   **Self-Regulation**: Connects to the owner's physiological and psychological state, self-regulating to provide optimal comfort and security.

在这个未来世界中，个人边界泡泡不仅是保护和控制的工具，更是一个动态、自主调节的个性化表达和互动平台。它让每个人能够通过边界泡泡定义和管理自己的虚拟存在，享受更加自由和个性化的数字生活。
In this future world, the personal boundary bubble is not only a tool for protection and control but also a dynamic, self-regulating platform for personalized expression and interaction. It allows everyone to define and manage their virtual presence through the boundary bubble, enjoying a more free and personalized digital life.

<img width="3143" alt="个人边界的构想" src="https://github.com/realitydeslab/playshop-summer-2024/assets/148893395/cf09fe80-c478-4a65-8358-a0cae85e622e">

<img width="1243" alt="截屏2024-07-01 11 34 27" src="https://github.com/realitydeslab/playshop-summer-2024/assets/148893395/8d0d867e-35ca-48e5-ab0e-ba36f3445a7b">

### **Your Reading List**
1. https://www.ribbonfarm.com/2015/01/16/on-the-design-of-escaped-realities/
2. https://www.bbc.com/news/technology-67865327
3. https://www.artanthropology.com/show.aspx?id=901&cid=7

###  **Visual Reference / Illustrations（视觉参考/插图）：**
![_ (21)](https://github.com/realitydeslab/playshop-summer-2024/assets/148893395/41598aeb-d3a0-4ab0-a411-2d392b0fa59b)
![Flatsheet membrane    Gas separation membrane_ flatsheet module_](https://github.com/realitydeslab/playshop-summer-2024/assets/148893395/6cd86c3f-f222-49b3-ba8e-3328ca7008d6)

**### References引用**：
v[1]“Latent Spaces: The High-Dimensional Infosphere Erik Lintunen erik.lintunen@network.rca.ac.uk Artist London, United Kingdom”
[2]Karen Barad. 2007. Meeting the Universe Halfway: Quantum Physics and the Entanglement of Matter and Meaning. Duke University Press, Durham, NC, 147.
[3]Venkatesh Rao. On the Design of Escaped Realities
[4] Duan Jianjun: Human beings exist entirely on the border - A brief introduction to Bakhtin’s humanistic thought
[5]Nancy Katherine Hayles. 2005. My Mother Was a Computer: Digital Subjects and Literary Texts. University of Chicago Press, Chicago, IL, 279–280.
[6] Alexander Lowen. 2005. The voice of the body. Bioenergetics Press.
[7]“NAMESPACES SEMIPERMEABLE MEMBRANES FOR APL APPLICATIONS” 
[8]“GZKP: A GPU Accelerated Zero-Knowledge Proof System” 
[9]Emotion Detection and Recognition using HRV Features Derived from Photoplethysmogram Signals 
[10]Avant-garde Iconographies of Combat: From the “Futurist Synthesis of War” to “Beat the Whites with the Red Wedge”Other Sections

**Comment by Axiuuuuu** [comment]
人对自身的认知是系统性的，对自身的接纳是整体性的。而机械部分必然需要进入到自我认知和认同当中，作为人的机械构成而被人接纳，否则人就会陷入分裂，陷入自我认知的矛盾中去。  所以在人的身体被技术覆盖到一定程度时，接受到的感官都来源于技术，被触摸被拥抱的还是自己的身体吗，此时身体的归属权是谁。People's understanding of themselves is systematic, and their acceptance of themselves is holistic. The mechanical part must enter into self-recognition and identification, and be accepted as the mechanical component of human beings. Otherwise, people will fall into division and contradictions in self-recognition.  Therefore, when a person's body is covered by technology to a certain extent, all the senses received come from technology. Is it still one's own body that is touched and embraced? Who owns the body at this time?

**Comment by botaohu** [comment]
https://www.ribbonfarm.com/2015/01/16/on-the-design-of-escaped-realities/

**Comment by glitch-kraft** [comment]
https://www.politicalcompass.org/test

**Comment by Axiuuuuu** [comment]
new：https://www.bbc.com/news/technology-67865327

**Comment by glitch-kraft** [comment]
Baarle-Hertog
Kaliningrad Oblast
Alaska
And other exclaves and enclaves

**Comment by botaohu** [comment]
programmablity of your personal boundary  -> bouncing ball (the consequence of violating the boundary) 
area of your personal space.

**Comment by glitch-kraft** [comment]
personal space
soft, hard, large, small, rigid, bouncing back, time-based....

**Comment by Axiuuuuu** [comment]
25词："Osmosis Boundaries: Your personal bubble in VR that adjusts for privacy, safety and social fun. Perceptually morphs to suit your needs for comfortable socialization!"

**Reply by glitch-kraft** [reply]
Osmosis Boundary. Your personal boundary bubble in VR, customised to your preferences for privacy, safety and social fun. Parametrically programmable, it adapts to your needs for comfortable socialising!

**Comment by Axiuuuuu** [comment]
100字摘要："Osmosis Boundary is the future of personal boundary bubbles and explores the concept of personal boundaries in virtual reality. It illustrates how these boundaries serve as mechanisms to protect privacy and security while enhancing personal expression and social interaction. By connecting with the user's physiological and psychological states, the boundary self-regulates to provide optimal comfort and safety. Key features include semi-permeable membranes, adaptable bubble shapes, multi-sensory permissions and zero-knowledge proofs that allow users to make informed, active choices without revealing their personal information. This innovative boundary concept ensures a secure yet interactive virtual experience that is tailored to individual needs.

**Comment by Axiuuuuu** [comment]
![Uploading 关键帧.png…]()

**Comment by glitch-kraft** [comment]
![image](https://github.com/realitydeslab/playshop-summer-2024/assets/46427283/b240d857-abe4-4e08-85b0-c955d364fce4)

**Reply by glitch-kraft** [reply]
клином красным бей белых

**Comment by glitch-kraft** [comment]
https://www.typeform.com/templates/t/market-research/
